src/test/java/example/LegacyFullTextIndexTest.java

```diff
@@ -22,14 +22,13 @@
     @Test
     public void shouldAllowIndexingAndFindingANode() throws Throwable
     {
-        // In a try-block, to make sure we close the driver after the test
-        try( Driver driver = GraphDatabase.driver( neo4j.boltURI() , Config.build().withEncryptionLevel( Config.EncryptionLevel.NONE ).toConfig() ) )
+        // In a try-block, to make sure we close the driver and session after the test
+        try(Driver driver = GraphDatabase.driver( neo4j.boltURI() , Config.build()
+                .withEncryptionLevel( Config.EncryptionLevel.NONE ).toConfig() );
+            Session session = driver.session() )
```

Comment:
Sorry @jexp you've lost me, that's exactly the same as my PR no? Perhaps the formatting is confusing?